packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -128,7 +129,9 @@ const Dropdown = React.forwardRef(function Dropdown(
   const ItemToElement = itemToElement;
   const toggleButtonProps = getToggleButtonProps();
   const helper = helperText ? (
-    <div className={helperClasses}>{helperText}</div>
+    <Text as="div" className={helperClasses}>
+      {helperText}
+    </Text>
```

Comment:
If I remember correctly, `helperText` could be a `PropTypes.node`, right?

Reply:
@joshblack Yeah, good point. It can also be a string. Do you think we should conditionally apply a `Text` based on if it's a string or not?